test/disk-stats-spike/run.sh

```
@@ -0,0 +1,98 @@
+  echo "FAIL(2): kubelet serves metrics but ZERO kubelet_volume_stats_* —"
+  echo "         the PVC storage backend has no kubelet MetricsProvider."
+  kubectl get pv -o jsonpath='{range .items[*]}{"  PV "}{.metadata.name}{" csi="}{.spec.csi.driver}{" hostPath="}{.spec.hostPath.path}{"\n"}{end}'
+  exit 1
+fi
+echo "  ok ($volstats_lines volume-stats series)"
+
+echo "== ASSERT (3): series carry namespace AND persistentvolumeclaim (both PVCs) =="
+sample="$(echo "$metrics" | grep '^kubelet_volume_stats_available_bytes' | grep 'namespace="db-a"' || true)"
+echo "$sample" | sed 's/^/    /'
+echo "$sample" | grep -q 'namespace="db-a"' || { echo "FAIL(3a): no namespace label"; exit 1; }
+echo "$sample" | grep -q 'persistentvolumeclaim="data"' && echo "$sample" | grep -q 'persistentvolumeclaim="config"' \
```

Shellcheck (0.11.0)

[info] Note that A && B || C is not if-then-else. C may run when A is true.

(SC2015)